snippets/card-product.liquid

```diff
@@ -366,9 +375,9 @@
                   >
                     <span>
                       {%- if card_product.selected_or_first_available_variant.available -%}
-                        {{ 'products.product.add_to_cart' | t }}
+                        ADD TO CART
```

Comment:
instead of relacing the dynamic text with your custom text, go into the translation file and update the text there.